### PR TITLE
added coffee

### PR DESCRIPTION
**Pull request by crazysmile11012**
#39

### CODE REVIEW

**Comment by crazysmile11012** [comment]
just fixed the conflict

File: CONTRIBUTING.md
Comment (Negative-light) [inline CONTRIBUTING.md:1]: This file looks good. We may need to do another round of edits on it later on as we do some development work in parrllel

File: README.md
Comment (Negative-light) [inline README.md:1]: There are still some spelling errors namely with Capitlization. Not a huge issue though

File: out/404.html
Comment (Negative-light) [inline out/404.html:1]: Please remove all of the ./out/* files

**Comment by crazysmile11012** [comment]
sorry i don't know how to only add 2 files into a pull instead of all (they where going to be merged anyways) lol
i know it is unclean sorry @Negative-light

File: src/app/about/page.tsx
Comment (Negative-light) [inline src/app/about/page.tsx:1]: These changes may cause a "merge conflict" need to see how we are going to fix that

File: src/app/page.tsx
Comment (Negative-light) [inline src/app/page.tsx:1]: Same thing possible issue for a merge conflict

**Review by Negative-light — CHANGES_REQUESTED** [review]
Overall good but please remove everything under ./out/*
This has been fixed in development and main by adding out back to the .gitignore so we probably will not have to deal with this in the future.

**Comment by Negative-light** [comment]
All good @crazysmile11012 we will have to educate ourselves on how to do a specialized merge. I am hoping we can get this fixed up

File: src/app/about/page.tsx
Comment (crazysmile11012) [reply src/app/about/page.tsx:1]: it would not build without me fixing charSet and className

File: src/app/page.tsx
Comment (crazysmile11012) [reply src/app/page.tsx:1]: i had to do charSet and className here too

File: out/404.html
Comment (crazysmile11012) [reply out/404.html:1]: ok

Comment (crazysmile11012) [reply out/404.html:1]: done

File: src/app/about/page.tsx
Comment (Negative-light) [reply src/app/about/page.tsx:1]: no that makes sense. Shouldn't be hard to handle the merge conflict

**Review by Negative-light — APPROVED** [review]
Looks good now. Just need to handle the Merge Conflict and it should be good

**Comment by Negative-light** [comment]
Well it solved itself

**Pull request merged**